mx.py

```diff
@@ -4282,7 +4282,7 @@ def compile(self, compilerArgs):
         if retcode:
             if _opts.verbose:
                 if _opts.very_verbose:
-                    raise subprocess.CalledProcessError(retcode, self.jdk.java + ' '.join(compilerArgs))
+                    raise subprocess.CalledProcessError(retcode, self.jdk.javac + ' ' + ' '.join(compilerArgs))
```

Comment:
Instead of throwing an exception, this should be:
```
    retcode = str(subprocess.CalledProcessError(retcode, self.jdk.javac + ' ' + ' '.join(compilerArgs)))
```

Reply:
Applied via 72083a4

Results can be seen at https://circleci.com/gh/neomatrix369/awesome-graal/426